packages/material-ui/src/Box/Box.js

```diff
@@ -1,17 +1,12 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import clsx from 'clsx';
-import { unstable_extendSxProp as extendSxProp } from '@material-ui/system';
-import styled from '../styles/experimentalStyled';
+import SystemBox from '@material-ui/system/Box';
+import { useTheme } from '../styles';
 
-const BoxRoot = styled('div', { muiName: 'MuiBox', skipVariantsResolver: true })``;
+const Box = React.forwardRef((props, ref) => {
```

Comment:
Not sure if it is ok that we are creating wrapper here, but we need to propagate default theme somehow. I am thinking it's maybe better if we just use the `experimentalStyled` here.. cc @oliviertassinari

Reply:
From what I have seen so far, extra components cost about 10% of performance in production. For a Box that is used very often, I doubt that it's negligible.

We can solve this problem in two different ways. We can either have a static property assigned to `SystemBox` but it would leak to every instance (maybe it's fine) or have a Box factory (cleaner).

Reply:
> or have a Box factory (cleaner)

+1 should be cleaner. We can compare this with previous implementation.

Reply:
Added with https://github.com/mui-org/material-ui/pull/26379/commits/9e756a479ee892afdec99465761970124fc023c6